ivy/functional/frontends/torch/__init__.py

```diff
@@ -3,6 +3,11 @@
 from .indexing_slicing_joining_mutating_ops import *
 from . import pointwise_ops
 from .pointwise_ops import *
+HEAD
```

Comment:
You should remove `HEAD` from Line 6, and the surrounding new lines. This came from when you merged from upstream, and should be removed. Also remove `+ upstream/master` from Line 49.